app/client/packages/design-system/widgets/src/components/TimeField/src/TimeField.tsx

```
@@ -0,0 +1,73 @@
+      >
+        {label}
+      </FieldLabel>
+      <Group className={textInputStyles.inputGroup}>
+        <DateInput
+          className={clsx(
+            textInputStyles.input,
+            dateTimeInputStyles.input,
+            getTypographyClassName("body"),
+          )}
+          data-date-input
+          data-size={Boolean(size) ? size : undefined}
```

Biome

[error] Avoid redundant `Boolean` call

It is not necessary to use `Boolean` call when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant `Boolean` call


(lint/complexity/noExtraBooleanCast)